images/scipy-ml-notebook/Dockerfile

```diff
@@ -61,10 +63,11 @@ RUN pip install --no-cache-dir  datascience \
 	pycocotools \
 	pillow \
 	nvidia-cudnn-cu11==8.6.0.163 \
-	tensorflow==2.12.* && \
+    tensorflow==2.12.0 \
+    keras==2.12.0 && \
```

Comment:
Had to pin TF to 2.12.0 since tests require equal versions and keras hasn't yet pushed 2.12.1.